wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -279,36 +281,34 @@ Features (osu!mania)
 
 | Button (Keyboard Shortcut) | Usage | Description |
 | -------------------------- | ----- | ----------- |
-| Whistle (`W`) | ("Select" on a Hit note) **Left-Click:** Use whistle sound on this note | Add a whistle sound addition to the selection. |
-| Finish (`E`) | ("Select" on a Hit note) **Left-Click:** Use finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
-| Clap (`R`) | ("Select" on a Hit note) **Left-Click:** Use clap sound on this note | Add a clap sound addition to the selection. |
+| Whistle (`W`) | ("Select" on a Hit note) **Left-Click:** Add a whistle sound on this note | Add a whistle sound addition to the selection. |
+| Finish (`E`) | ("Select" on a Hit note) **Left-Click:** Add a finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
+| Clap (`R`) | ("Select" on a Hit note) **Left-Click:** Add a clap sound on this note | Add a clap sound addition to the selection. |
 
 #### Assist Tools
 
 | Button (Keyboard Shortcut) | Usage | Description |
 | -------------------------- | ----- | ----------- |
-| Grid Snap (`T`) | **`Shift` (Hold):** Temporary toggle. **`Ctrl` (Hold):** Temporary disable angle snapping. | Hit notes will be snapped to the grid by default. |
-| Distance Snap (`Y`) | **`Alt` (Hold):** Temporary toggle. Switch Beat Snap Divisor to Distance Snap **`Alt` + Mousewheel:** Adjust Distance Snap multiplier. | Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused. |
 | Lock Notes (`L`) | ("Select" on a Hit note) **Left-Click:** Lock this note | Lock selected note to current position and timestamp. |
 
 For **keysound** (adding SE on hitnotes), select the note and **`Alt` + Left Click** on the selected note (a pop-up window will be shown with sound samples inside your beatmap folder). Refer to "Sampling" section at end of page.
 
 ### Bottom (Song Timeline)
 
-![Song Timeline](/wiki/shared/BE_STL.jpg "Song Timeline")
+![Song's Timeline](img/BE_HSAT.jpg "Song's Timeline")
 
-At **bottom-left**, it shows **timestamp in milliseconds (ms)** and **song duration %**.
+At the **bottom-left**, there is the **timestamp** in milliseconds (ms) and **song duration** in percentage. The percentage will switch to "intro" or "outro" if there is storyboarding before or after the music.
 
-At **bottom-centre**, it shows the **timeline with markings and the compulsory music player buttons.** For "Test" button, your beatmap will be saved to test the beatmap starting on current timestamp.
+At the **bottom-centre**, there is a **timeline** with markings and the compulsory music player buttons. The `Test` button to the right of the timeline will save your beatmap then enter play test mode, starting from the current timestamp.
 
 #### Bookmarking command
```

Comment:
obsolete section you should exclude features that work identically in both versions of the editor